### PR TITLE
♻️ be more selective about which docs to lazy render

### PR DESCRIPTION
**Pull request by dok**

### CODE REVIEW

**Comment by domharrington** [comment]
@erunion think this is good to go! @kanadgupta can you work together to build a release for this?

File: packages/api-explorer/package-lock.json [hunk 1]
```diff
@@ -11320,30 +11317,20 @@
       },
       "dependencies": {
         "ajv": {
-          "version": "6.10.0",
```
Comment (erunion) [inline packages/api-explorer/package-lock.json]: @dok Did you mean to downgrade ajv here?

Comment (dok) [reply packages/api-explorer/package-lock.json]: I did not intend to downgrade. The update to `package-lock.json` was a result of running `npx lerna bootstrap`.

Comment (erunion) [reply packages/api-explorer/package-lock.json]: K cool, just checking. Looks like 5.5.2 falls in line with what the `react-jsonschema-form` dependency is requesting anyways so 👍

**Pull request merged**